### PR TITLE
SLT-95 resource optimisation

### PR DESCRIPTION
**Pull request by floretan**
I did some testing, and if we have more resources for the post-release job (see https://github.com/wunderio/drupal-project-k8s/pull/99) we can reduce the requested memory for the other containers. 

As discussed in Slack, I would also recommend removing the readiness checks. They are creating a considerable load on the service by themselves and prevent us from being able to lower the resources without the checks failing due to timeouts. They have also been responsible for a few confusing issues for developers, in the end we are better off without them.

Note that the deployment is a bit slower because in this branch the post-release job doesn't yet get additional resources.

### CODE REVIEW

File: chart/templates/drupal-deployment.yaml [hunk 1]
```diff
@@ -33,13 +33,6 @@ spec:
         livenessProbe:
           tcpSocket:
             port: 9000
-        readinessProbe:
-          exec:
-            command:
-            - /bin/sh
-            - -c
-            - if [ ! {{ include "drupal.deployment-in-progress-test" . }} ]; then timeout -t 3 drush check-bootstrap; else exit 1; fi
```
Comment (Jancis) [inline chart/templates/drupal-deployment.yaml]: I believe that test is still needed so we don't get the tables created while post-deployment doing initial `drush site-install`.

Comment (floretan) [reply chart/templates/drupal-deployment.yaml]: You are right. We can skip `drush check-bootstrap` but should still check for the deployment in progress.

File: chart/templates/drupal-deployment.yaml [hunk 2]
```diff
@@ -35,11 +35,7 @@ spec:
             port: 9000
         readinessProbe:
           exec:
-            command:
-            - /bin/sh
-            - -c
-            - if [ ! {{ include "drupal.deployment-in-progress-test" . }} ]; then timeout -t 3 drush check-bootstrap; else exit 1; fi
-          periodSeconds: 3
+            command: ['/bin/sh', '-c', 'if [ {{ include "drupal.deployment-in-progress-test" . }} ]; then exit 1; fi']
```
Comment (ragnarkurmwunder) [inline chart/templates/drupal-deployment.yaml]: To be more verbose I suggest:
`/bin/sh -x -c ...`
Extra log saves time when things go wrong.

Comment (Jancis) [reply chart/templates/drupal-deployment.yaml]: Good suggestion @ragnarkurmwunder . I believe this issue might be in several places, could you take a look at the code and make a PR for it, please? I'll try to bring it over to https://github.com/wunderio/charts when we test if on k8s

Comment (ragnarkurmwunder) [reply chart/templates/drupal-deployment.yaml]: Actually, you can simplify it too :)
```
{{ include "drupal.deployment-in-progress-test" . }} && exit 1
```

**Pull request merged**